src/ui/sass/modules/_AlternativeVerticals.scss

```diff
@@ -13,7 +13,7 @@ $alternative-verticals-emphasized-font-weight: var(--yxt-font-weight-medium) !de
   background-color: var(--yxt-color-brand-white);
   border: var(--yxt-border-default);
 
-  @media (min-width: var(--yxt-breakpoint-mobile-max)) {
```

Comment:
Are there other media queries that need to be similarly updated?

Reply:
there are not-- i took a look at all other media queries